### PR TITLE
Add sign out functionality #20

### LINKED ISSUE #20: View - homepage
On the user page there's a button that after clicking it sign out user from current session.

### PR DESCRIPTION
**Pull request by kamildebicki**
Closes #20

### CODE REVIEW

**Comment by student-tomasz** [comment]
I don't really see a point in creating this home page. We don't have. For a signed user it's `wishlists#index` of his wishlists and for not signed user it's the sign in/sign up page. It's already done when adding authorization logic in #60.

**Comment by student-tomasz** [comment]
Although we do need the log out link in `views/layouts/application`.

**Comment by kamildebicki** [comment]
My bad, I don't check final version of prototypes of views for our app.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <%= javascript_include_tag 'application' %>
   </head>
   <body>
+    <% if user_signed_in? %>
+    <%= link_to 'Sign out', destroy_user_session_path, :method => :delete %>
```
Comment (student-tomasz) [inline app/views/layouts/application.html.erb]: One tab more.

**Comment by student-tomasz** [comment]
👍

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (agalkiewicz) [inline app/views/layouts/application.html.erb]: You can write `method: :delete`.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -7,6 +7,9 @@
...
+    <% end %>
```
Comment (agalkiewicz) [inline app/views/layouts/application.html.erb:12]: IMO you can add empty line after this one.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -7,6 +7,9 @@
...
     <%= flash_messages %>
```
Comment (agalkiewicz) [inline app/views/layouts/application.html.erb:14]: And after this one. But it's only my opinion.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,6 +7,9 @@
     <%= javascript_include_tag 'application' %>
   </head>
   <body>
+    <% if user_signed_in? %>
+      <%= link_to 'Sign out', destroy_user_session_path, :method => :delete %>
```
Comment (student-tomasz) [inline app/views/layouts/application.html.erb]: Powtórzę za Alicję: Popraw ten hash na nowy syntax.

Comment (Kacper3331) [inline app/views/layouts/application.html.erb]: Also I think you should use double quotation marks

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -7,6 +7,11 @@
     <%= javascript_include_tag 'application' %>
   </head>
   <body>
+
```
Comment (student-tomasz) [inline app/views/layouts/application.html.erb]: Remove this empty line. Optionally add one below `flash_message`.

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -13,6 +13,7 @@
     <% end %>
```
Comment (agalkiewicz) [inline app/views/layouts/application.html.erb:12]: Line 10 isn't removed.

**Comment by student-tomasz** [comment]
👍

**Comment by agalkiewicz** [comment]
:+1:

**Comment by gotar** [comment]
squash, and you can merge (but having any test for this will be nice)

**Pull request merged**